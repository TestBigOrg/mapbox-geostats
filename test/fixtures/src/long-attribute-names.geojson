{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {
        "LoremipsumdolorsitametconsecteturadipisicingelitseddoeiusmodtemporincididuntutlaboreetdoloremagnaaliquaUtenimadminimveniamquisnostrudexercitationullamcolaborisnisiutaliquipexeacommodoconsequatDuisauteiruredolorinreprehenderitinvoluptatevelitessecillumdoloreeufugiatnullapariaturExcepteursintoccaecatcupidatat": 1
      },
      "geometry": {
        "type": "Point",
        "coordinates": [1, 1]
      }
    },
    {
      "type": "Feature",
      "properties": {
        "LoremipsumdolorsitametconsecteturadipisicingelitseddoeiusmodtemporincididuntutlaboreetdoloremagnaaliquaUtenimadminimveniamquisnostrudexercitationullamcolaborisnisiutaliquipexeacommodoconsequatDuisauteiruredolorinreprehenderitinvoluptatevelitessecillumdoloreeufugiatnullapariaturExcepteursintoccaecatcupidatat": 2
      },
      "geometry": {
        "type": "Point",
        "coordinates": [1, 1]
      }
    }
  ]
}
