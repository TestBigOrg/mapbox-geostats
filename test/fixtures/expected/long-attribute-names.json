{
  "layerCount": 1,
  "layers": [
    {
      "layer": "long-attribute-names",
      "count": 2,
      "attributeCount": 1,
      "attributes": [
        {
          "values": [
            1,
            2
          ],
          "count": 2,
          "attribute": "LoremipsumdolorsitametconsecteturadipisicingelitseddoeiusmodtemporincididuntutlaboreetdoloremagnaaliquaUtenimadminimveniamquisnostrudexercitationullamcolaborisnisiutaliquipexeacommodoconsequatDuisauteiruredolorinreprehenderitinvoluptatevelitessecillumdolor",
          "type": "Number",
          "min": 1,
          "max": 2
        }
      ],
      "geometry": "Point"
    }
  ]
}
